src/components/Navbar/Navbar.container.tsx

```diff
@@ -64,15 +64,25 @@ const SidebarContainer = ({ sx, menuItems }: Props) => {
         }
     };
 
-    return (
-        <SidebarView
-            sx={sx}
+    const handlePageSelect = (name: string) => {
+        setActivePage(name);
+    };
+
+    return mobile ? (
+        <NavbarMobile
+            activePage={activePage}
+            onPageSelect={handlePageSelect}
+            menuItems={menuItems}
+            logout={handleLogout}
+        />
+    ) : (
+        <NavbarView
             activePage={activePage}
-            setActivePage={setActivePage}
+            onPageSelect={handlePageSelect}
             menuItems={menuItems}
             logout={handleLogout}
```

Comment:
I think you could assign the proper component to a variable and then use it once with the props.
```
const NavbarComponent = mobile ? NavbarMobile : NavbarView;

<NavbarComponent ... ... ... />
```

Reply:
Done